api/test/wfl/unit/sink_test.clj

```diff
@@ -25,11 +25,13 @@
   (let [inputs (resources/read-resource "sarscov2_illumina_full/inputs.edn")
         dataset (resources/read-resource "sarscov2-illumina-full-inputs.json")
         result (sink/rename-gather-bulk (UUID/randomUUID) dataset "flowcell" inputs {:flowcell_tgz "flowcell_tgz"
-                                                                                     :flowcell_id "flowcell_id"})]
+                                                                                     :flowcell_id "flowcell_id"
+                                                                                     :sample_rename_map "sample_rename_map"})]
```

Comment:
Is this a fileref, where is this established if so?

Reply:
Sorry, back to this. Yes, it is a fileref. The dataset under `sarscov2-illumina-full-inputs.json` contains the table schema for `flowcell` that shows this field as a fileref field. The inputs did not make use of it, so I picked it to add a test for null handling on the fileref columns.